### DISCUSSION TITLE
Sample Code for Kotlin/JS

### DISCUSSION THREAD

**Discussion by joerg-rade** [pdvrieze/xmlutil#17] [Q&A]
I need to deserialize some XML (bootstrap3.xsd) in Kotlin/JS.

Is there any sample code?

**Comment by pdvrieze** [comment]
The test suite should help you https://github.com/pdvrieze/xmlutil/blob/master/serialization/src/commonTest/kotlin/nl/adaptivity/xml/serialization/TestCommon.kt 

I'm not sure that deserializing xml schema is easy though. In any case you will probably want to look at the annotations that allow more precise specification of the used names. One element you want to consider is to enable the autopolymorphic option (it is a new option that actually makes life easier - of course you still need to declare the module).
